src/react/components/DeviceInfoCard.tsx

```diff
@@ -18,22 +18,41 @@ const DeviceInfoCard: React.FC<{ deviceInfo: Device }> = ({ deviceInfo }) => {
     <div className="device-info">
       <h2>Selected Device Info</h2>
       <ul>
-        <li><strong>Serial:</strong> {deviceInfo.serial}</li>
-        <li><strong>Model:</strong> {deviceInfo.model}</li>
-        <li><strong>IP:</strong> {deviceInfo.ip}</li>
-        <li><strong>Android Version:</strong> {deviceInfo.androidVersion}</li>
-        <li><strong>SDK Version:</strong> {deviceInfo.sdkVersion}</li>
         <li>
```

Comment:
This multiline makes me crazy!!!! i'll refactor to minor components as the project get more atomically organazed.
😩
Already did som here: https://github.com/victorwads/RookieNode/commit/a90183deeb7b561b23997393df2a6926b8b782e3#diff-3d49f4e5a5adcdb84fdd01c136e6a64bfa452d5df0665e727a9551b656262739L10-R146